src/GNNGraphs/query.jl

```diff
@@ -273,6 +273,15 @@ or a vector.
                 Default `true`.
 
 """
```

Comment:
you are stealing the docstring of the method defined for GNNGraph. This method should have its own docstring.